documentation/prettier/readme.md

```diff
@@ -0,0 +1,57 @@
+# Prettier Config Documentation
+
+> Documentation for the `@strv/prettier-config` module.
+
+## Usage
+
+### Installing
+
+Run:
+
+```bash
+# Yarn:
+yarn add --dev @strv/prettier-config
+
+# npm:
+npm install --save-dev @strv/prettier-config
+```
+
+And reference it in your `package.json` file:
+
+```js
+{
+  // ...
+  "prettier": "@strv/prettier-config"
+}
+```
+
+_Note: If you are using `prettier` < v1.17 check [this note](#prettier-117)._
+
+### <a id="extending"></a> Extending
+
+To extend the current options you will need to create a `.prettierrc.js` and add the following:
+
+```js
+module.exports = {
+  ...require('@strv/prettier-config'),
+
+  // Add custom options bellow:
+  useTabs: true,
+}
+```
+
+## Notes
+
+### <a id="prettier-117"></a> Prettier < v1.17.x
+
+Versions of `prettier` prior to v1.17 did not feature the possibility to use a shared config via `package.json`. The way to do it is similar to when [extending](#extending), except that you can just export the config directly:
+
+```js
+// .prettierrc.js
+
+module.exports = require('@strv/prettier-config')
```

Comment:
I do not fully understand the difference between the _preferred_ way of extending the config and this pre-1.17 way - functionally they are identical except that in the former you also use destructuring to extend the config.

Perhaps one of those examples is not right...? 🤔

Reply:
Well, there isn't really a difference :sweat_smile: You have to fallback to the "old" way (using a _prettierrc.js_) if you want to extend a shared config. 

Maybe they are not organized the best way? I agree that extending is usually an exception so I moved it out from "Usage". Any other suggestions? :thinking:

Reply:
The updated description is now much clearer (ie. I understood the difference 😇), I think it may stay in its current form. Thanks! 👍